packages/terra-embedded-content-consumer/src/EmbeddedContentConsumer.module.scss

```diff
@@ -1,5 +1,13 @@
+// Themes
+@import './clinical-lowlight-theme/EmbeddedContentConsumer.module';
+@import './orion-fusion-theme/EmbeddedContentConsumer.module';
+
 :local {
   .visually-hidden-text {
     height: 0;
   }
+
+  .iframe-focus-style {
+    outline: var(--terra-embedded-content-consumer-focus-outline, 2px dashed #000);
```

Comment:
Don't you also need an outline-offset?  Did UX not specify one in this case?

Reply:
I wasn't aware of that `outline-offset`. I was not working with UX on this. @cm9361 do you know who I can reach out to? 

cc: @scottwilmarth

Reply:
We have been working with @eawww (Eric Wilson)

Reply:
I worked with @eawww on this and we've come up with something that would work.